app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if visitor["vaccine"].get("expiration_date") < datetime.date.today():
+            raise OutdatedVaccineError("Outdated vaccine")
+        if not visitor.get("wearing_a_mask"):
+            raise NotWearingMaskError("All visitors should wear masks")
+        return f"Welcome to {self.name}"

```

Comment:
It's great that you have used type annotations in your function definition. However, it's a good practice to also specify the return type of the function. In this case, it would be `-> str`.